### ISSUE TITLE
play audio on Alexa pyhsical device

### ISSUE THREAD

**Issue by mduzgun** [thorsten-gehrig/alexa-remote-control#52]
Is there a way play audio on Alexa pyhsical device by using alexa-remote-control ?

**Comment by adn77** [comment]
Apart from Amazon Music and TuneIn, I know of a hack that redirects the DNS of a TuneIn station to a local shoutcast stream. Unfortunately that requires some serious engineering ;)

**Comment by mduzgun** [comment]
My problem is that I want to send audio url which is maybe my voice, to pyhsical Alexa device and I want to hear voice from Alexa device. I can send text and hear as voice my device, but I search how to directly audio file url and play it.

**Comment by Apollon77** [comment]
Then you need to have an audio player output via bluetooth and connect your Alexa device as bluetooth player with this computer. I had this working for some time ... not ideal but working. No other ways known beside the already told

**Comment by mduzgun** [comment]
yes, via bluetooth, I can use Alexa as speaker, but this does not solve my problem :(

**Comment by Apollon77** [comment]
Have a raspi or such with  Bluetooth chip, connect as audio player to Alexa and then platy locally your audio file and it will be output on Alexa ... so you need to cntrol playing the audio file

**Comment by noureddin212** [comment]
Have you tryed MyMedia ?

Le jeu. 16 mai 2019 à 15:06, Ingo Fischer <notifications@github.com> a
écrit :
